### PR TITLE
Credit Score

### PR DESCRIPTION
**Pull request by NunoMartins21**
# Credit Score

image

## Type of PR

- [x] - feature

## Description

Created Credit Score feature

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -5,7 +5,7 @@ root = true
 charset = utf-8
 indent_style = space
 indent_size = 2
-insert_final_newline = true
+# insert_final_newline = true
```
Comment (kbooz) [inline .editorconfig:8]: why? this is necessary on some cases

File: apps/website/components/templates/credit-score/LoadingModal.tsx
Comment (kbooz) [inline apps/website/components/templates/credit-score/LoadingModal.tsx:1]: use i18n

File: apps/website/components/templates/credit-score/credit-score.tsx
Comment (kbooz) [inline apps/website/components/templates/credit-score/credit-score.tsx:1]: can't we break down this file?
it has 600+ lines!

File: apps/website/pages/protocol/credentials/[id]/show.tsx
Comment (kbooz) [inline apps/website/pages/protocol/credentials/[id]/show.tsx:1]: unecessary changes

**Review by kbooz — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (NunoMartins21) [reply .editorconfig]: Fixed. Idk why this was removed @Siddharth9890

**Review by kbooz — APPROVED** [review]

**Pull request merged**